MAINTAINERS.md

```diff
@@ -0,0 +1,46 @@
+This document outlines some processes that the maintainers should stick to.
+
+## Issues Process
+
+There are 3 types of issues that can be created:
+
+- `"bug`
+- `"new rule"`
+- `"enhancement"`
+
+### Triage
+
+The triage process is basically making sure that the issue is correctly reported (and ask for more info if not), and categorize it correctly if it belongs to a different type than initially assigned.
+
+- When a new issue is created, they'll include a `"triage"` label
+- If the issue is correctly reported, please remove the `"triage"` label, so we know is valid and ready to be tackled
+- If the issue is **not** correctly reported, please ask for more details and add the `"awaiting response"` label, so we know more info has been requested to the author
+- If the issue belong to an incorrect category, please update the labels to put it in the right category
+- If the issue is duplicated, please close it including a comment with a link to the duplicating issue
+
+## Pull Requests Process
+
+### Main PR workflow
+
+_TODO: pending to describe the main PR process_
```

Comment:
I'm pretty sure you'll find tons of things to add here and in other areas of the document. Please, let's add that in a separate PR since I only wanted to document the Triage process somewhere, and I added some additional info that came to my mind.